drivers/sensor/openrocket/imu/openrocket_imu.h

```diff
@@ -0,0 +1,49 @@
+#ifndef OPENROCKET_IMU_H
+#define OPENROCKET_IMU_H
+
+#include "openrocket_sensors.h"
+
+#include <stdbool.h>
+
+enum axis {
+    X,
+    Y,
+    Z,
+};
```

Comment:
Can you make this more explicit? Something like OPEN_ROCKET_AXIS_X or something?